Ginger/GingerCoreNET/ValueExpressionLib/ValueExpression.cs

```diff
@@ -1467,21 +1441,24 @@ private void ReplaceRegExWithValue(string p, string[] a)
             }
         }
 
+
         private void ReplaceVarWithValue(string p, string[] a)
         {
             string VarName = null;
             string tmp = mValueCalculated;
 
             string suba = string.Join(" ", new ArraySegment<string>(a, 1, a.Length - 1));
-
+            GetExtraParam(p);
             string[] ParamVal = suba.Split('=');
             {
                 string ParamName = ParamVal[0];
                 string Val = suba.Substring(suba.IndexOf('=') + 1);
                 switch (ParamName)
                 {
                     case "Name":
-                        VarName = Val;
+                        {
+                            VarName = RemoveExtraParamFromName(Val);
+                        }
                         break;
                     default:
                         // TODO err unknown param
```

Comment:
Add code to pull the Value of "Name"( dic[key].value) for pulling the matched variable